src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughLastLineCommentCheck.java

```diff
@@ -29,7 +29,7 @@ public void method2(int i) {
         switch (i) {
             case 1:
                 i++;
-                /* block */ /* fallthru */ // comment
+                /* block */ /* Fallthru */ // comment
```

Comment:
What is reason of such updates?
If no good reason let's not change, we usually not changing old test data .
But we can change violation comments.

Reply:
@romani Done, and removed violation